src_modules/player/index.ts

```diff
@@ -84,6 +73,11 @@ export default class PlayerModule extends Module {
         return queue;
     }
 
+    public popQueue(guild: Guild) {
+        const queue = this.getQueue(guild);
```

Comment:
This leads to an error. Change this to use the same way of getting queue as in the clearQueue function below, using the guild's id.